metis-enrichment/metis-enrichment-service/src/main/java/eu/europeana/enrichment/service/RedisInternalEnricher.java

```diff
@@ -222,16 +232,20 @@ private void loadEntity(EntityType entityType, MongoTerm term, Jedis jedis) {
               parents.toArray(new String[]{}));
         }
         if (termList.getOwlSameAs() != null) {
-          for (String sameAs : termList.getOwlSameAs()) {
-            jedis.hset(entityType.cachedEntityPrefix + CACHED_SAMEAS, sameAs, term.getCodeUri());
-          }
+          loadSameAs(entityType, term, jedis, termList.getOwlSameAs());
```

Comment:
It's a bit inconsistent to have a small private method for this while similar operations are done inline. I assume the issue here is the indentation level (too deep nesting)? May I suggest undoing this change and instead move the try-catch block to the calling method (loadEntities())?

Reply:
Yeah that makes more sense